src/plugins/profiling/server/routes/stacktrace.ts

```diff
@@ -172,7 +172,7 @@ export async function mgetStackTraces(
   await logExecutionLatency(logger, 'processing data', async () => {
     // flatMap() is significantly slower than an explicit for loop
     for (const res of stackResponses) {
-      for (const trace of res.body.docs) {
+      for (const trace of res.docs) {
```

Comment:
I received an error when viewing the stack traces (aka Traces button). When adding `body` back, the response return successfully.

Reply:
See https://github.com/optimyze/kibana/pull/66#discussion_r859505796